templates/shared/forms/interactive/osm-contact-us.html

```diff
@@ -0,0 +1,254 @@
+            <label for="infrastructure-preferences-public-cloud">Public cloud</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="infrastructure-preferences-openstack" name="infrastructure-preferences">
+            <label for="infrastructure-preferences-openstack">OpenStack</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="infrastructure-preferences-vmware" name="infrastructure-preferences">
+            <label for="infrastructure-preferences-vmware">VMware</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="infrastructure-preferences-other" name="infrastructure-preferences">
+            <label for="infrastructure-preferences-other">Other</label>
+          </div>
+        </div>
+      </div>
+
+      <div class="js-formfield">
+        <h3 class="p-heading--five">What are the priority use cases? (optional)</h3>
+        <div class="row u-no-padding">
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-private-lte/5g-network" name="priority-cases">
+            <label for="priority-cases-private-lte/5g-network">Private LTE/5G network</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-ims" name="priority-cases">
+            <label for="priority-cases-ims">IMS</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-packet-core" name="priority-cases">
+            <label for="priority-cases-packet-core">Packet core</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-subscriber-data-management" name="priority-cases">
+            <label for="priority-cases-subscriber-data-management">Subscriber data management</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-ran-oran" name="priority-cases">
+            <label for="priority-cases-ran-oran">RAN / ORAN</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-oss-bss" name="priority-cases">
+            <label for="priority-cases-oss-bss">OSS/BSS</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-transport" name="priority-cases">
+            <label for="priority-cases-transport">Transport</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-general-it-use-case" name="priority-cases">
+            <label for="priority-cases-general-it-use-case">General IT use case</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="priority-cases-other" name="priority-cases" value="Other">
```

Comment:
same for this and all the other "other" options

Reply:
The same as what?

Reply:
us the specific "other containter" mentioned [here](https://github.com/canonical-web-and-design/ubuntu.com/pull/10068#discussion_r677531279)

Reply:
I see, thanks